### ISSUE TITLE
check_smtp raises an arguments error after the update via watchdog

### ISSUE THREAD

**Issue by mnin** [mailcow/mailcow-dockerized#4619]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/mailcow/mailcow-dockerized/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... I understand that not following the below instructions will result in immediate closure and/or deletion of my issue.
- [X] ... I have understood that this bug report is dedicated for bugs, and not for support-related inquiries.
- [X] ... I have understood that answers are voluntary and community-driven, and not commercial support.
- [X] ... I have verified that my issue has not been already answered in the past. I also checked previous [issues](https://github.com/mailcow/mailcow-dockerized/issues).

### Description

After updating today to the release with the tag `2022-06` got some watchdog notifications:

Maybe the `nagios-plugins-smtp` package got an update in the new Alpine version.

### Logs

```bash
check_smtp: Set either -s/--ssl or -S/--starttls
Usage:
check_smtp -H host [-p port] [-4|-6] [-e expect] [-C command] [-R response] [-f from addr]
[-A authtype -U authuser -P authpass] [-w warn] [-c crit] [-t timeout] [-q]
[-F fqdn] [-S] [-L] [-D warn days cert expire[,crit days cert expire]] [--sni] [-v]
```

### Steps to reproduce

1. Just enable the watchdog

**Comment by ro78** [comment]
I think I have the same issue. I deleted the email, I'll confirm if I receive it again.

**Comment by DerLinkman** [comment]
On hold until nagios released a new fix.

**Comment by MAGICCC** [comment]
closed as current version 2.4.2 in watchdog container and no issues anymore
